docs/deployment.md

```diff
@@ -105,17 +105,20 @@ FROM node:alpine AS runner
 WORKDIR /app
 
 ENV NODE_ENV production
-RUN addgroup -g 1001 -S nodejs
-RUN adduser -S nextjs -u 1001
-USER nextjs
-EXPOSE 3000
 
 # You only need to copy next.config.js if you are NOT using the default configuration
 # COPY --from=builder /app/next.config.js ./
 COPY --from=builder /app/public ./public
 COPY --from=builder /app/.next ./.next
 COPY --from=builder /app/node_modules ./node_modules
 
+RUN addgroup -g 1001 -S nodejs
+RUN adduser -S nextjs -u 1001
+RUN chown -R nextjs:nodejs /app/.next
```

Comment:
Should we give ownership to the entire `/app` directory?

Reply:
@armand1m @yordis I'd love your opinion here!

Reply:
I think that sounds reasonable

Reply:
Thanks @leerob @armand1m looks like it was merged, but if we want to make that change then we need to change it to `RUN chown -R nextjs:nodejs /app`

Reply:
@leerob somehow I failed to see this notification

```docker
COPY --chown=nextjs:nodejs
```

I think you meant to do that instead of moving the statement after the copy but it seems to be the same